[2024-06-08T08:39:12.458+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T08:39:12.488+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_t1.upload_test1 manual__2024-06-08T07:11:19.232646+00:00 [queued]>
[2024-06-08T08:39:12.503+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_t1.upload_test1 manual__2024-06-08T07:11:19.232646+00:00 [queued]>
[2024-06-08T08:39:12.504+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 4
[2024-06-08T08:39:12.523+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_test1> on 2024-06-08 07:11:19.232646+00:00
[2024-06-08T08:39:12.532+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3711) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-08T08:39:12.534+0000] {standard_task_runner.py:63} INFO - Started process 3715 to run task
[2024-06-08T08:39:12.532+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_t1', 'upload_test1', 'manual__2024-06-08T07:11:19.232646+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/Etl_Dag.py', '--cfg-path', '/tmp/tmp33m08g63']
[2024-06-08T08:39:12.538+0000] {standard_task_runner.py:91} INFO - Job 74: Subtask upload_test1
[2024-06-08T08:39:12.609+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_t1.upload_test1 manual__2024-06-08T07:11:19.232646+00:00 [running]> on host 6c00989aad97
[2024-06-08T08:39:12.736+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_t1' AIRFLOW_CTX_TASK_ID='upload_test1' AIRFLOW_CTX_EXECUTION_DATE='2024-06-08T07:11:19.232646+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-08T07:11:19.232646+00:00'
[2024-06-08T08:39:12.738+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T08:39:12.771+0000] {logging_mixin.py:188} INFO - prepare to upload csv file at transform folder
[2024-06-08T08:39:12.771+0000] {logging_mixin.py:188} INFO - transform
[2024-06-08T08:39:12.782+0000] {logging_mixin.py:188} INFO - {'timeStamp': '20240608163910', 'filePath': '//opt/***/dags/../local/20240608163910_combined.csv'}
[2024-06-08T08:39:12.795+0000] {base.py:84} INFO - Using connection ID 's3_conn' for task execution.
[2024-06-08T08:39:12.797+0000] {connection_wrapper.py:387} INFO - AWS Connection (conn_id='s3_conn', conn_type='aws') credentials retrieved from extra.
[2024-06-08T08:39:13.407+0000] {logging_mixin.py:188} INFO - 20240608163910
[2024-06-08T08:39:13.407+0000] {logging_mixin.py:188} INFO - 20240608
[2024-06-08T08:39:13.408+0000] {logging_mixin.py:188} INFO - 20240608/transform/20240608163910_combined.csv
[2024-06-08T08:39:13.408+0000] {logging_mixin.py:188} INFO - //opt/***/dags/../local/20240608163910_combined.csv
[2024-06-08T08:39:13.409+0000] {logging_mixin.py:188} INFO - file exist ???:True
[2024-06-08T08:39:13.410+0000] {logging_mixin.py:188} INFO - uploading file to s3 now...
[2024-06-08T08:39:13.444+0000] {logging_mixin.py:188} INFO - file uploded
[2024-06-08T08:39:13.445+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T08:39:13.446+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T08:39:13.457+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_t1, task_id=upload_test1, run_id=manual__2024-06-08T07:11:19.232646+00:00, execution_date=20240608T071119, start_date=20240608T083912, end_date=20240608T083913
[2024-06-08T08:39:13.514+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T08:39:13.536+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T08:39:13.538+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
